src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java

```diff
@@ -386,15 +388,16 @@ private DefaultClockSnapshotModel(ClockModel clock) {
 		protected boolean isRunning;
 	}
 
-	public static class UpdateClockTimerTask extends TimerTask {
+	protected static class UpdateClockTimerTask extends TimerTask implements ScoreBoardListener {
 		public static final String PROPERTY_INTERVAL_KEY = DefaultClockModel.class.getName() + ".interval";
 		private static long update_interval = DefaultClockModel.CLOCK_UPDATE_INTERVAL;
 
 		public UpdateClockTimerTask() {
 			try {
 				update_interval = Integer.parseInt(ScoreBoardManager.getProperty(PROPERTY_INTERVAL_KEY));
 			} catch ( Exception e ) { }
-			startSystemTime = System.currentTimeMillis();
+			startSystemTime = scoreBoardClock.getCurrentTime();
+			scoreBoardClock.addScoreBoardListener(this);
```

Comment:
Currently all listens are via the DefaultScoreBoardModel, so this should work the same way for consistency.